src/formatDate.js

```diff
@@ -50,7 +50,53 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const toSeperator = toFormat[toFormat.length - 1];
+  const fromSeperator = fromFormat.pop();
+  const dateArray = date.split(fromSeperator);
+  let dateYear, dateMonth, dateDay;
```

Comment:
Declare each variable on the separate line.
```suggestion
  let dateYear;
  let dateMonth;
  let dateDay;
```

Reply:
Need to fix this as well.